src/background/custom-filters.js

```diff
@@ -28,6 +28,7 @@ import Options from '/store/options.js';
 import CustomFilters from '/store/custom-filters.js';
 
 import { setup, reloadMainEngine } from '/background/adblocker.js';
+import CustomContentBlocks from '/store/custom-content-blocks';
```

Comment:
please add `.js` - can we have a linter rule for it?

Reply:
It's my IDE, I need to check if I can force adding `.js` in the IDE, and will also look if there is an ESLint rule for that.

Reply:
Ok, my IDE has an option for that ;)

Reply:
For eslint we would need to add additional plugin, so I think it is not worth it.